index.html

```diff
@@ -0,0 +1,196 @@
+            <div class="back-link2"><div class="link2">css</div></div>
+            <div class="back-link3"><div class="link3">javaScript</div></div>
+            </div>
+            <button class="see-project1"><div class="see-project">See Project</div></button>
+            </section>
+        </div>
+        <section>
+            <div></div>
+            <div></div>
+        </section>
+    </section>
+    <section class="section4">
+        <div>
+            <section class="border-box">
+                <img src="images/Snapshoot Portfolio (2).png" class="snapshot">
+                <p class="head2">Tonic</p>
+                <div class="up1">
+                    <div class="p1">CANOPY</div>
+                    <div class="counter"></div>
+                    <div class="p2">Back End Dev</div>
+                    <div class="counter"></div>
+                    <div class="p3">2015</div>
+                </div>
+            </section>
+            <img src="" alt="">
+            <section>
+            <p class="p41">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="middle">
+            <div class="back-link1"><div class="link1">html</div></div>
+            <div class="back-link2"><div class="link2">css</div></div>
+            <div class="back-link3"><div class="link3">javaScript</div></div>
+            </div>
+            <button class="see-project1"><div class="see-project">See Project</div></button>
+            </section>
+        </div>
+        <section>
+            <div></div>
+            <div></div>
+        </section>
+    </section>
+    <section class="section5">
+        <div>
+            <section class="border-box">
+                <img src="images/Snapshoot Portfolio (3).png" class="snapshot">
+                <p class="head2">Multi-Post Stories</p>
+                <div class="up">
+                    <div class="p1">CANOPY</div>
+                    <div class="counter"></div>
+                    <div class="p2">Back End Dev</div>
+                    <div class="counter"></div>
+                    <div class="p3">2015</div>
+                </div>
+            </section>
+            <img src="" alt="">
+            <section>
+            <p class="p4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="middle">
+            <div class="back-link1"><div class="link1">html</div></div>
+            <div class="back-link2"><div class="link2">css</div></div>
+            <div class="back-link3"><div class="link3">javaScript</div></div>
+            </div>
+            <button class="see-project1"><div class="see-project">See Project</div></button>
+            </section>
+        </div>
+        <section>
+            <div></div>
+            <div></div>
+        </section>
+    </section>
+</div> 
+<section class="section6">
+<h1 class="h">About <br>Myself</h1> 
```

Comment:
- [ ] Since `h1` is already used, Please make sure that you use any of the header tag `h2` or `h3` for semantic reasons :pray: